infrastructure/repository/labels-workflow.tf

```diff
@@ -125,6 +125,10 @@ variable "workflow_labels" {
       color       = "ac72f0", # color:terraform (link on black)
       description = "Introduces a new service."
     },
+    "no-changelog" = {
```

Comment:
In case we want to be consistent with Terraform core, that project uses a `no-changelog-needed` label.

https://github.com/hashicorp/terraform/pulls?q=is%3Apr+is%3Aopen+label%3Ano-changelog-needed